tests/Coduo/PHPHumanizer/Tests/NumberTest.php

```diff
@@ -166,6 +179,28 @@ public function binarySuffixDataProvider()
     /**
      * @return array
      */
+    public function preciseBinarySuffixDataProvider()
+    {
+        return array(
+            array(-1, -1, 3),
+            array("0 bytes", 0, 3),
+            array("1 bytes", 1, 3),
+            array("1.000 kB", 1024, 3),
+            array("1.001 kB", 1025, 3),
+            array("1.500 kB", 1536, 3),
+            array("5.000 MB", 1048576 * 5, 3),
+            array("2.000 GB", 1073741824 * 2, 3),
+            array("3.000 TB", 1099511627776 * 3, 3),
+            array("1.178 PB", 1325899906842624, 3),
```

Comment:
Could you please add some tests for precision that is bigger and lower than 3? For example, 1,2,3,4,5,6,7  ?